src/create/helpers/base-contracts.ts

```diff
@@ -0,0 +1,10 @@
+// Import the base contracts'

```

Comment:
Does this need to be a separate helper file - seems like these are both one line helpers and are only being used once or twice in the other file anyway

Reply:
if we are going to do some github checking now it makes sense for it to be helper file to me